404 when attempting to install the package

Chocolatey failed to install Soundnode on my desktop so I checked the logs to find this:

``` 
2016-09-10 23:31:24,490 [INFO ] -  Attempt to get headers for http://www.soundnodeapp.com/downloads/win/Soundnode-App.zip failed.
2016-09-10 23:31:24,491 [INFO ] -    Exception calling "GetResponse" with "0" argument(s): "The remote server returned an error: (404) Not Found."
2016-09-10 23:31:24,492 [INFO ] -  Downloading soundnode-app 32 bit
2016-09-10 23:31:24,492 [INFO ] -    from 'http://www.soundnodeapp.com/downloads/win/Soundnode-App.zip'
2016-09-10 23:31:24,840 [ERROR] -  Exception calling "GetResponse" with "0" argument(s): "The remote server returned an error: (404) Not Found."
2016-09-10 23:31:24,841 [ERROR] -  At C:\ProgramData\chocolatey\helpers\functions\Get-WebFile.ps1:81 char:3
```

I checked the download link for Windows on the website and it looks like Chocolatey uses the wrong link.

--- a/soundnode-app/soundnode-app.nuspec
+++ b/soundnode-app/soundnode-app.nuspec
@@ -3,7 +3,7 @@
   <metadata>
     <id>soundnode-app</id>
     <title>Soundnode App</title>
-    <version>0.6.4.20160426</version>
+    <version>0.6.5</version>
     <authors>Michael Lancaster</authors>
     <owners>Jose V. Trigueros</owners>
     <summary>Soundcloud desktop client</summary>
@@ -16,12 +16,12 @@
     <packageSourceUrl>https://github.com/jvtrigueros/chocolatey-packages/soundnode-app</packageSourceUrl>
     <docsUrl>https://github.com/Soundnode/soundnode-app/wiki</docsUrl>
     <bugTrackerUrl>https://github.com/Soundnode/soundnode-app/issues</bugTrackerUrl>
-    <projectSourceUrl>https://github.com/Soundnode/soundnode-app/tree/0.6.4</projectSourceUrl>
+    <projectSourceUrl>https://github.com/Soundnode/soundnode-app/tree/0.6.5</projectSourceUrl>
     <tags>soundnode-app soundnode soundcloud</tags>
     <copyright/>
-    <licenseUrl>https://github.com/Soundnode/soundnode-app/blob/0.6.4/LICENSE.md</licenseUrl>
+    <licenseUrl>https://github.com/Soundnode/soundnode-app/blob/0.6.5/LICENSE.md</licenseUrl>
     <requireLicenseAcceptance>false</requireLicenseAcceptance>
-    <iconUrl>https://cdn.rawgit.com/Soundnode/soundnode-app/0.6.4/app/soundnode.png</iconUrl>
+    <iconUrl>https://cdn.rawgit.com/Soundnode/soundnode-app/0.6.5/app/soundnode.png</iconUrl>
     <releaseNotes>https://github.com/Soundnode/soundnode-app/releases</releaseNotes>
   </metadata>
   <files>

--- a/soundnode-app/tools/chocolateyinstall.ps1
+++ b/soundnode-app/tools/chocolateyinstall.ps1
@@ -28,7 +28,7 @@ New-Item "$target.ignore" -type file -force | Out-Null
 $zipFileList.AppendLine("$target.ignore") | Out-Null
 
 # Get icon from repository
-Get-ChocolateyWebFile -packgeName $packageName -fileFullPath "$toolsDir\soundnode.ico" -url $iconUrl
+Get-ChocolateyWebFile -packageName $packageName -fileFullPath "$toolsDir\soundnode.ico" -url $iconUrl
 $zipFileList.AppendLine("$toolsDir\soundnode.ico") | Out-Null
 
 # Create shortcut in Programs Start Menu

--- a/soundnode-app/tools/chocolateyinstall.ps1
+++ b/soundnode-app/tools/chocolateyinstall.ps1
@@ -6,7 +6,7 @@ $packageName  = 'soundnode-app'
 $shortcutPath = "C:\ProgramData\Microsoft\Windows\Start Menu\Programs\Soundnode App.lnk"
 $toolsDir     = "$(Split-Path -parent $MyInvocation.MyCommand.Definition)"
 $target       = Join-Path $toolsDir $exe
-$url          = 'http://www.soundnodeapp.com/downloads/win/Soundnode-App.zip'
+$url          = 'http://www.soundnodeapp.com/downloads/win/Soundnode.zip'
 $zipFileList  =  New-Object System.Text.StringBuilder
 $zipLogPath   = Join-Path $env:ChocolateyPackageFolder "${packageName}Install.zip.txt"
 
@@ -16,7 +16,7 @@ $packageArgs = @{
   url           = $url
 
   registryUninstallerKey = 'soundnode-app'
-  checksum      = '3873B3DD5175421BFA4FFDE0D1779EDD'
+  checksum      = '3b17d7e5c454b711eddc33b802016174'
   checksumType  = 'md5'
 }
 

